pyproject.toml

```diff
@@ -9,6 +9,7 @@ python = "^3.9"
 attrs = "^21.2.0"
 requests = "^2.26.0"
 structlog = "^21.1.0"
+smart-open = {extras = ["s3"], version = "^5.2.1"}
```

Comment:
Not a big deal or necessary, but I've been trying to keep the dependencies in alphabetical order because it's easier for humans to scan. Unfortunately a manual process as poetry (and pipenv, sadly) don't do this automatically.